installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -36,7 +36,7 @@ include::modules/installation-bare-metal-agent-installer-config-yaml.adoc[levelo
 
 include::modules/validations-before-agent-iso-creation.adoc[leveloffset=+1]
 
-include::modules/agent-install-ipi-install-root-device-hints.adoc[leveloffset=+1]
+include::modules/agent-install-root-device-hints.adoc[leveloffset=+1]
```

Comment:
@zaneb the file names are internal and can be left as is.

Reply:
:+1: I agree it's not a problem, I just renamed it to reduce my own confusion.